itou/utils/faker_providers.py

```diff
@@ -13,6 +14,10 @@ def asp_ea2_filename(self, date: datetime.date = None) -> str:
         date_part = random.randint(0, 99999999) if date is None else date.strftime("%Y%m%d")
         return f"FLUX_EA2_ITOU_{date_part}.zip"
 
+    def geiq_filename(self, date: datetime.date = timezone.localdate()) -> str:
```

Comment:
A éviter idéalement (même si ici pour des tests, cela n'a pas de conséquence): la date est fixée une fois pour toute pour le processus python. Donc si le programme tourne sur plusieurs jours, tu auras toujours comme date par défaut, la date au lancement du programme.
D'habitude, c'est plutôt le classique `date=None` / `if date is None: date = timezone.localdate()`